rootfs/network/parse-config

```diff
@@ -34,7 +34,7 @@ echo "$CONFIG_FILE" >/tmp/config_file
 parse_config()
 {
     local config="$(
-        sed -n "/^$1[ \t]*=[ \t]*/{s///;s/[ \t]*$//;p}" <"$CONFIG_FILE")"
+        sed -n "/^[ \t]*$1[ \t]*=[ \t]*/{s///;s/[ \r\t]*$//;p}" <"$CONFIG_FILE")"
```

Comment:
It might be slightly confusing to allow a mix of `\r` and other whitespace characters at the end of the line.  I think I'd suggest changing `[ \r\t]*$` to `[ \t]*\r*$`, and treat `\r` in any other place as malformed.  It's probably excess pedantry to replace `\r*` with `\r?`.

Reply:
The file that was causing an error was line terminated with \r\r\n. What windows craziness... I'll make the other change